fuse_core/include/fuse_core/throttled_callback.h

```diff
@@ -57,14 +57,15 @@ class ThrottledCallback
   /**
    * @brief Constructor
    *
-   * @param[in] keep_callback   The callback to call when kept, i.e. not dropped. Defaults to nullptr
-   * @param[in] drop_callback   The callback to call when dropped because of the throttling. Defaults to nullptr
-   * @param[in] throttle_period The throttling period duration in seconds. Defaults to 0.0, i.e. no throttling
-   * @param[in] use_wall_time   Whether to use ros::WallTime or not. Defaults to false
+   * @param[in] keep_callback       The callback to call when kept, i.e. not dropped. Defaults to nullptr
+   * @param[in] drop_callback       The callback to call when dropped because of the throttling. Defaults to nullptr
+   * @param[in] throttle_period     The throttling period duration in seconds. Defaults to 0.0, i.e. no throttling
+   * @param[in] use_wall_time       Whether to use ros::WallTime or not. Defaults to false
```

Comment:
Why the extra indentation?
```suggestion
   * @param[in] keep_callback   The callback to call when kept, i.e. not dropped. Defaults to nullptr
   * @param[in] drop_callback   The callback to call when dropped because of the throttling. Defaults to nullptr
   * @param[in] throttle_period The throttling period duration in seconds. Defaults to 0.0, i.e. no throttling
   * @param[in] use_wall_time   Whether to use ros::WallTime or not. Defaults to false
```